libs/dkg/include/dkg/rbc.hpp

```diff
@@ -83,22 +79,19 @@ class RBC
   struct Party
   {
     std::unordered_map<TagType, std::bitset<sizeof(MsgType) * 8>>
-                                    flags;  ///< Marks for each message tag what messages have been received
-    uint8_t                         deliver_s;        ///< Counter for messages delivered
+            flags;          ///< Marks for each message tag what messages have been received
+    uint8_t deliver_s = 1;  ///< Counter for messages delivered - initialised to 1
```

Comment:
why 1?

Reply:
It seems it's a feature of the RBC protocol